client/src/main/java/org/apache/celeborn/client/ShuffleClient.java

```diff
@@ -175,6 +177,20 @@ public abstract RssInputStream readPartition(
 
   public abstract void shutDown();
 
+  // Write data to a specific map partition, input data's type is Bytebuf.
+  // data's type is Bytebuf to avoid copy between application and netty
+  // closecallback will do some clean opertions like memory release.
+  public abstract int pushData(
```

Comment:
Better to rename to pushDataToLocation

Reply:
pushData is changed to pushDataToLocation